libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/TestOptions.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.Dialogs.Adaptive
+{
+    /// <summary>
+    /// Test options.
+    /// </summary>
+    public class TestOptions
+    {
+        /// <summary>
+        /// Kind.
+        /// </summary>
+        public const string Kind = "conversation.testOptions";
+
+        /// <summary>
+        /// Gets or sets random seed.
+        /// </summary>
+        /// <value>
+        /// Random seed.
+        /// </value>
+        [JsonProperty(PropertyName = "randomSeed")]
+        public int RandomSeed { get; set; }
+
+        /// <summary>
+        /// Gets or sets random value.
+        /// </summary>
+        /// <value>
+        /// Random value.
+        /// </value>
+        [JsonProperty(PropertyName = "randomValue")]
+        public int RandomValue { get; set; }
+    }
```

Comment:
> } [](start = 3, length = 2)

For C# you want to capture overflow properties

[JsonExtendedData]
public IDictionary<string, object> Properties{get;set;}